cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -55,6 +55,18 @@ export default class SortableTablePo extends ComponentPo {
     return cy.getId('sortable-table-promptRemove');
   }
 
+  deactivateButton() {
+    return cy.getId('sortable-table-deactivate');
+  }
+
+  activateButton() {
+    return cy.getId('sortable-table-activate');
+  }
+
+  downloadYamlButton() {
```

Comment:
This one is generic, but at the `ResourceTable` level

Reply:
Gotcha, moved to resource-table.po.ts